.taskcluster.yml

```diff
@@ -118,211 +118,208 @@ tasks:
-                                                ${action.description}
-
-                                                Action triggered by clientID `${clientId}`
+                                        kind: 'action-callback'
                                     else:
-                                        name: "Decision Task for cron job ${cron.job_name}"
-                                        description: 'Created by a [cron task](https://firefox-ci-tc.services.mozilla.com/tasks/${cron.task_id})'
-                      provisionerId: "mobile-${level}"
-                      workerType: "decision"
-                      tags:
-                          $if: 'tasks_for in ["github-push", "github-pull-request"]'
-                          then:
-                              kind: decision-task
-                          else:
-                              $if: 'tasks_for == "action"'
-                              then:
-                                  kind: 'action-callback'
-                              else:
-                                  $if: 'tasks_for == "cron"'
-                                  then:
-                                      kind: cron-task
-                      routes:
-                          $flattenDeep:
-                              - checks
-                              - $if: 'level == "3"'
-                                then:
-                                    - tc-treeherder.v2.${project}.${head_sha}
-                                    - $if: 'tasks_for == "github-push"'
-                                      then:
-                                          - index.mobile.v2.${project}.branch.${short_head_branch}.latest.taskgraph.decision
-                                          - index.mobile.v2.${project}.branch.${short_head_branch}.revision.${head_sha}.taskgraph.decision
-                                          - index.mobile.v2.${project}.revision.${head_sha}.taskgraph.decision
-                                    - $if: 'tasks_for == "cron"'
-                                      then:
-                                          # cron context provides ${head_branch} as a short one
-                                          - index.mobile.v2.${project}.branch.${head_branch}.latest.taskgraph.decision-${cron.job_name}
-                                          - index.mobile.v2.${project}.branch.${head_branch}.revision.${head_sha}.taskgraph.decision-${cron.job_name}
-                                          - index.mobile.v2.${project}.branch.${head_branch}.revision.${head_sha}.taskgraph.cron.${ownTaskId}
-                      scopes:
-                          # `https://` is 8 characters so, ${repoUrl[8:]} is the repository without the protocol.
-                          $if: 'tasks_for == "github-push"'
-                          then:
-                              $let:
-                                  short_head_branch:
```

Comment:
This let statement has been removed because `short_head_branch` is now defined up above.